apps/application-system/api/src/environments/environment.ts

```diff
@@ -17,26 +17,13 @@ const devConfig = {
     clientLocationOrigin: `http://localhost:${
       process.env.WEB_FRONTEND_PORT ?? '4242'
     }/umsoknir`,
-    emailOptions: {
-      useTestAccount: !(process.env.USE_SES === 'true'),
-      useNodemailerApp: process.env.USE_NODEMAILER_APP === 'true' ?? false,
-      options: {
-        region: process.env.EMAIL_REGION ?? 'eu-west-1',
-      },
-    },
```

Comment:
Kristófer is changing those same lines in this PR https://github.com/island-is/island.is/pull/15493 maybe two should make sure this won't break anything

Reply:
Good eye. Looks to me that my changes cover what Kristófer is trying to do. We will have a merge conflict though and need to handle that when the time comes.